ganttproject/src/net/sourceforge/ganttproject/ChartTabContentPanel.java

```diff
@@ -62,6 +62,9 @@ public void windowOpened(WindowEvent windowEvent) {
       }
     });
   }
+  void replaceImagePanel(Image image){
+    myImagePanel.setIcon(new ImageIcon(image.getScaledInstance(-1, 50, Image.SCALE_DEFAULT)));
```

Comment:
any reason why the height must be 50px? There are calculations in the code below which take into account DPI setting. They probably shall not be ignored?